src/TextInput/TextInput.tsx

```diff
@@ -3,27 +3,26 @@ import styled from 'styled-components'
 import { defaultStyledProps, globalInput, InputProps } from '../utils/styled'
 import { Omit } from '../utils/types'
 
-type OmitOnChange = Omit<InputHTMLAttributes<HTMLInputElement>, 'onChange'>
+export type OmitOnChangeHTMLInputAttributes = Omit<InputHTMLAttributes<HTMLInputElement>, 'onChange'>
 
-interface TextInputProps extends InputProps {
+export interface TextInputProps extends InputProps {
   value: string
   onChange: (value: string, event: ChangeEvent<HTMLInputElement>) => void
 }
 
-export const PlainTextInput: React.FC<TextInputProps & OmitOnChange> = ({ onChange, ...rest }) => (
-  <input
+export const TextInput: React.FC<TextInputProps & OmitOnChangeHTMLInputAttributes> = ({ onChange, ...rest }) => (
+  <StyledInput
     type='text'
-    onChange={(event) => onChange(event.target.value, event)}
+    onChange={(event: any) => onChange(event.target.value, event)}
     {...rest}
   />
 )
 
-export const TextInput = styled(PlainTextInput)`
+export const StyledInput = styled.input <TextInputProps> `
```

Comment:
also nit: `styled.input<TextInputProps>`

Reply:
> Also, there's an error I also see in the stories all the time. Not sure why it's not picking up the defaultProps?

What error - i dont see one.

Reply:
and codestyle taught me that style!

Reply:
> > Also, there's an error I also see in the stories all the time. Not sure why it's not picking up the defaultProps?
> 
> What error - i dont see one.

Travis